### PR TITLE
Fix the error in name of the secret variable

### PR DESCRIPTION
**Pull request by kostazol**
Fix the error in name of the secret variable.

### CODE REVIEW

File: .env.local.example [hunk 1]
```diff
@@ -1,5 +1,5 @@
 NEXTAUTH_URL=http://localhost:3000
-NEXTAUTH_SECRET​= # Linux: `openssl rand -hex 32` or go to https://generate-secret.now.sh/32
+SECRET= # Linux: `openssl rand -hex 32` or go to https://generate-secret.now.sh/32
```
Comment (lluia) [inline .env.local.example:2]: This is not an error, it is a new environment variable we introduced 💭
https://next-auth.js.org/configuration/options#nextauth_secret

**Comment by lluia** [comment]
@kostazol are you ok if I close the PR? I think you got confused about the env variable 🤔

File: .env.local.example (hunk 1 above)
Comment (kostazol) [reply .env.local.example:2]: ok, than change it please in https://github.com/nextauthjs/next-auth-example/blob/main/pages/api/examples/jwt.ts

Comment (kostazol) [reply .env.local.example]: I changed it.

**Review by balazsorban44 — APPROVED** [review]

**Pull request merged**